src/plugins/dropdown/dropdown-fr.hbs

```diff
@@ -0,0 +1,210 @@
+		<li><button type="button">Autre chose</button></li>
+	</ul>
+</div>
+<details class="mrgn-tp-md">
+	<summary>Source code</summary>
+	<pre><code>&lt;div class="dropdown"&gt;
+	&lt;button class="btn btn-default dropdown-toggle" type="button"&gt;Menu déroulant &lt;span class="caret"&gt;&lt;/span&gt;&lt;/button&gt;
+	&lt;ul class="dropdown-menu"&gt;
+		&lt;li&gt;&lt;button type="button"&gt;Action&lt;/button&gt;&lt;/li&gt;
+		&lt;li&gt;&lt;button type="button"&gt;Autre action&lt;/button&gt;&lt;/li&gt;
+		&lt;li&gt;&lt;button type="button"&gt;Autre chose&lt;/button&gt;&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;</code></pre>
+</details>
+
+<h2>Dimensions</h2>
+<div class="dropdown">
+	<button class="btn btn-default btn-lg dropdown-toggle" type="button">Menu déroulant <span class="caret"></span></button>
+	<ul class="dropdown-menu">
+		<li><a href="#">Un lien</a></li>
+		<li><a href="#">Un autre lien</a></li>
+		<li><a href="#">Autre chose</a></li>
+	</ul>
+</div>
+<div class="dropdown">
+	<button class="btn btn-default btn-sm dropdown-toggle" type="button">Menu déroulant <span class="caret"></span></button>
+	<ul class="dropdown-menu">
+		<li><a href="#">Un lien</a></li>
+		<li><a href="#">Un autre lien</a></li>
+		<li><a href="#">Autre chose</a></li>
+	</ul>
+</div>
+<div class="dropdown">
+	<button class="btn btn-default btn-xs dropdown-toggle" type="button">Menu déroulant <span class="caret"></span></button>
+	<ul class="dropdown-menu">
+		<li><a href="#">Un lien</a></li>
+		<li><a href="#">Un autre lien</a></li>
+		<li><a href="#">Autre chose</a></li>
+	</ul>
+</div>
+<details class="mrgn-tp-md">
+	<summary>Source code</summary>
+	<pre><code>&lt;div class="dropdown"&gt;
+	&lt;button class="btn btn-default btn-lg dropdown-toggle" type="button"&gt;Menu déroulant &lt;span class="caret"&gt;&lt;/span&gt;&lt;/button&gt;
+	&lt;ul class="dropdown-menu"&gt;
+		&lt;li&gt;&lt;a href="#"&gt;Un lien&lt;/a&gt;&lt;/li&gt;
+		&lt;li&gt;&lt;a href="#"&gt;Un autre lien&lt;/a&gt;&lt;/li&gt;
+		&lt;li&gt;&lt;a href="#"&gt;Autre chose&lt;/a&gt;&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;
+&lt;div class="dropdown"&gt;
+	&lt;button class="btn btn-default btn-sm dropdown-toggle" type="button"&gt;Menu déroulant &lt;span class="caret"&gt;&lt;/span&gt;&lt;/button&gt;
+	&lt;ul class="dropdown-menu"&gt;
+		&lt;li&gt;&lt;a href="#"&gt;Un lien&lt;/a&gt;&lt;/li&gt;
+		&lt;li&gt;&lt;a href="#"&gt;Un autre lien&lt;/a&gt;&lt;/li&gt;
+		&lt;li&gt;&lt;a href="#"&gt;Autre chose&lt;/a&gt;&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;
+&lt;div class="dropdown"&gt;
+	&lt;button class="btn btn-default btn-xs dropdown-toggle" type="button"&gt;Menu déroulant &lt;span class="caret"&gt;&lt;/span&gt;&lt;/button&gt;
+	&lt;ul class="dropdown-menu"&gt;
+		&lt;li&gt;&lt;a href="#"&gt;Un lien&lt;/a&gt;&lt;/li&gt;
+		&lt;li&gt;&lt;a href="#"&gt;Un autre lien&lt;/a&gt;&lt;/li&gt;
+		&lt;li&gt;&lt;a href="#"&gt;Autre chose&lt;/a&gt;&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;</code></pre>
+</details>
+
+<h2>Couleurs</h2>
```

Comment:
```suggestion
<h2>Menu déroulant basé sur un état</h2>
```

We can't rely on colour. We need to rely on the meaning provided by that use of some specific colour.